src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -242,6 +262,35 @@ module Make (Inputs : Inputs_intf) :
     List.bind (successors t breadcrumb) ~f:(fun succ ->
         succ :: successors_rec t succ )
 
+  let compute_new_successor_root_hash t parent_node =
```

Comment:
the `successor_root_hash` of a node is the same for all successors, so I find the word `new` here a little misleading as to what this function actually does.